projects/addon-commerce/components/input-card/input-card.component.ts

```diff
@@ -59,7 +58,7 @@ export class TuiInputCardComponent extends AbstractTuiInputCard<string> {
     }
 
     get formattedCard(): string {
```

Comment:
This is only used in template, we can use maskito pipe.

Reply:
You are right, I've forgot about this pipe))

Reply:
fixed